float64[3] rpyAng
float64[3] rpyAngVel
float64[3] rpyAngAcc
float64 msgTime